Alexa.NET.Tests/IntentTests.cs

```diff
@@ -35,7 +35,7 @@ public void Complex_intent_single_property()
 
 			Assert.Equal("AMAZON", name.Namespace);
 			Assert.Equal("SearchAction", name.Action);
-            Assert.Equal(1,name.Properties.Count);
+            Assert.True(name.Properties.Count == 1);
```

Comment:
It's good to be tidying up the assertions, could we remove the logic and use Assert.Single instead?

Reply:
Assert.Single would be the best solution